platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -47,97 +33,80 @@ class AboutModal extends Component {
+        {` `}
+        <a
+          className="btn btn-default"
+          target="_blank"
+          rel="noopener noreferrer"
+          href="https://github.com/OHIF/Viewers/issues/new/choose"
+        >
+          {t('Report an issue')}
+        </a>
+        {` `}
+        <a
+          className="btn btn-default"
+          target="_blank"
+          rel="noopener noreferrer"
+          href="http://ohif.org"
+        >
+          {t('More details')}
+        </a>
+      </div>
+      <div>
+        <h3>{t('Version Information')}</h3>
+        <table className="table table-responsive">
+          <thead>
+            <tr>
+              <th>{t('Name')}</th>
+              <th>{t('Value')}</th>
+            </tr>
+          </thead>
+          <tbody>{itemsPreset().map(item => renderTableRow(item))}</tbody>
+        </table>
+      </div>
+    </Modal.Body>
+  );
+};
 
-  render() {
-    const { t } = this.props;
-    return (
-      <Modal
-        show={this.props.isOpen}
-        onHide={this.props.onCancel}
-        aria-labelledby="ModalHeader"
-        className="AboutModal modal fade themed in"
-        backdrop={false}
-        large={true}
-        keyboard={false}
-      >
-        <Modal.Header closeButton>
-          <Modal.Title>{t('OHIF Viewer - About')}</Modal.Title>
-        </Modal.Header>
-        <Modal.Body data-cy="about-modal">
-          <div className="btn-group">
-            <a
-              className="btn btn-default"
-              target="_blank"
-              href="https://groups.google.com/forum/#!forum/cornerstone-platform"
-            >
-              {t('Visit the forum')}
-            </a>
-            {` `}
-            <a
-              className="btn btn-default"
-              target="_blank"
-              href="https://github.com/OHIF/Viewers/issues/new/choose"
-            >
-              {t('Report an issue')}
-            </a>
-            {` `}
-            <a
-              className="btn btn-default"
-              target="_blank"
-              href="http://ohif.org"
-            >
-              {t('More details')}
-            </a>
-          </div>
-          <div>
-            <h3>{t('Version Information')}</h3>
-            <table className="table table-responsive">
-              <thead>
-                <tr>
-                  <th>{t('Name')}</th>
-                  <th>{t('Value')}</th>
-                </tr>
-              </thead>
-              <tbody>
-                {this.itemsPreset().map(item => this.renderTableRow(item))}
-              </tbody>
-            </table>
-          </div>
-        </Modal.Body>
-      </Modal>
-    );
-  }
-}
+AboutModal.propTypes = {
+  t: PropTypes.func.isRequired,
+};
 
-const connectedComponent = withTranslation('AboutModal')(AboutModal);
-export { connectedComponent as AboutModal };
-export default connectedComponent;
+const ConnectedComponent = withTranslation('AboutModal')(AboutModal);
+export { ConnectedComponent as AboutModal };
```

Comment:
I "think" we can use `useTranslation` from `react-i18n` if we're using a function component? @rodrigolabs might know better.

Reply:
Does this still need to be a ConnectedComponent? What's it grabbing from Redux?

Reply:
That would also allow us to ditch `ConnectedComponent`

This technically isn't a "Modal" anymore. It's just "About OHIF" content, that we'll jam into the body of a modal at some point. We can probably shift it out of the UI project.

Reply:
Yes, we can use `useTranslation` in a function component, like this:

```javascript
import { useTranslation } from 'react-i18next';

const { t } = useTranslation('AboutModal'); // --> namespace
return <>{t('Browser')}</>
```